src/cool_seq_tool/mappers/exon_genomic_coords.py

```diff
@@ -484,6 +522,35 @@ async def _genomic_to_transcript_exon_coordinate(
 
         params = {key: None for key in TranscriptExonData.model_fields}
 
+        if is_fusion_transcript_segment:
+            # Check if is_fusion_transcript_segment is given
+            mane_transcripts = self.mane_transcript_mappings.get_gene_mane_data(gene)
+            transcript = mane_transcripts[0]["RefSeq_nuc"]
+            alt_ac = self.sr.chromosome_to_acs(chromosome)[0][0]
```

Comment:
This is a quick review so didn't check... is `chromosome` validated before? If not, you should not assume that there will be a response from `chromosome_to_acs`

Reply:
@jarbesfeld I was thinking of something like this (if chromosome is not validated)
```suggestion
            alt_acs, w = self.sr.chromosome_to_acs(chromosome)
            if not alt_acs:
            ....
```